### PR TITLE
Fix links

### PR DESCRIPTION
**Pull request by BrainonSilicon**
Fixed broken links to contributing guidelines

### CODE REVIEW

**Review by eirini-zormpa — COMMENTED** [review]
I'm not sure how to check that the links are working now 🙈 If I go to the fix links branch from Code I get a 404?

File: README.md [hunk 1]
```diff
@@ -46,4 +46,4 @@ A massive thank you goes to these wonderful people who have contributed to and b
 
 <!-- ALL-CONTRIBUTORS-LIST:END -->
 
-This project follows the [all-contributors](https://github.com/all-contributors/all-contributors) specification. Please read our [Guidelines](.contributing) on how to contribute to this Glossary. The Glossary is maintained and built by our community so if you believe a term is missing, please feel welcome to add or suggest content. For information on how to do this, read through the [Contributing](./contributing.md) information.
+This project follows the [all-contributors](https://github.com/all-contributors/all-contributors) specification. Please read our [Guidelines](docs/contributing.md) on how to contribute to this Glossary. The Glossary is maintained and built by our community so if you believe a term is missing, please feel welcome to add or suggest content. For information on how to do this, read through the [Contributing](docs/contributing.md) information.
```
Comment (eirini-zormpa) [inline README.md:49]:
```suggestion
This project follows the [all-contributors](https://github.com/all-contributors/all-contributors) specification. Please read our [Contributors Guidelines](docs/contributing.md) on how to contribute to this Glossary. The Glossary is maintained and built by our community so if you believe a term is missing, please feel welcome to add or suggest content.
```

**Review by eirini-zormpa — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (eirini-zormpa) [reply README.md:49]: I thought it'd be nice to keep the "Contributors Guidelines" file name consistent and I removed some content that seemed to me like repetition.
Looking at the files, I can see that the path is correct but is there a way to clink on the link while it's in a PR and make sure it takes you to the right place?

Comment (BrainonSilicon) [reply README.md:49]: ......it worked locally 🤣 

Okay being serious though I tend to go on the repo on that branch and then click through to check that it worked (that was how I figured out they were broken to begin with)

**Comment by BrainonSilicon** [comment]
Okay I may be being daft but to changes from 

`docs/contributing.md` to `./contributing` works and both point to the correct document. 

@eirini-zormpa: Is the `./` being clever on github because I always thought it pointed to the exact subdirectory that that file was in.

**Comment by eirini-zormpa** [comment]
that's what I thought too!

File: README.md [hunk 2]
```diff
@@ -7,15 +7,15 @@
 ---
 The AIM Research Support Facility (RSF), based at The Alan Turing Institute with collaborators from Swansea University and the University of Edinburgh, offers AI and advanced data science support to the research teams part of the [NIHR's Artificial Intelligence for Multiple Long-Term Conditions (AIM)](https://www.nihr.ac.uk/blog/artificial-intelligence-to-understand-clusters-of-multiple-long-term-conditions-an-nihr-priority/25171) project.
 
-Our community has compiled a [Glossary](./glossary) of common terms relating to Health, Medicine, Artificial Intelligence, and Data Science to support the development and dissemnation of AIM project work. The Glossary can be accessed by visiting the public website which has been built with JupytherBook and deployed using GitHub Pages. </br>
+Our community has compiled a [Glossary](docs/glossary) of common terms relating to Health, Medicine, Artificial Intelligence, and Data Science to support the development and dissemnation of AIM project work. The Glossary can be accessed by visiting the public website which has been built with JupytherBook and deployed using GitHub Pages. </br>
```
Comment (eirini-zormpa) [inline README.md:10]:
```suggestion
Our community has compiled a [Glossary](docs/glossary.md) of common terms relating to Health, Medicine, Artificial Intelligence, and Data Science to support the development and dissemnation of AIM project work. The Glossary can be accessed by visiting the public website which has been built with JupytherBook and deployed using GitHub Pages. </br>
```

**Comment by eirini-zormpa** [comment]
I prefer the docs/ syntax as it is less ambiguous!

The way I checked the link the first time was also by going to the branch and clicking on it, but I got a 404 - that was because the glossary file was missing the .md extension. Fixed that now so we're good to go :rocket:

**Pull request merged**